js/sdk/src/actyx.ts

```diff
@@ -163,3 +188,60 @@ export const Actyx = {
     }
   },
 }
+
+/**
+ * "Reference counter" for living Actyx
+ */
+const actyxInstanceRegister = (() => {
+  type AppId = AppManifest['appId']
+  type Version = AppManifest['version']
+  type Signature = AppManifest['signature']
+  type ActyxHost = ActyxOpts['actyxHost']
+  type ActyxPort = ActyxOpts['actyxPort']
+  type Param = [AppId, Version, Signature, ActyxHost, ActyxPort]
```

Comment:
I dislike the `Param` name, `Params`/`Parameters` makes much more sense given it's a list.

I'd also add a brief docstring

Reply:
these types aren’t visible to anyone, so a docstring won’t be displayed anywhere — did you mean a comment?